src/modules/cookie-preference/cookie-preference.test.jsdom.js

```diff
@@ -0,0 +1,85 @@
+import createCookiePreference from './index';
+import jsCookies from '../../../node_modules/js-cookie/src/js.cookie';
+
+describe('Cookie Preference', () => {
+    it('should set a preference in a cookie', () => {
+        const cookiePreference = createCookiePreference('testCookie', ['colour', 'drink', 'food']);
+        cookiePreference.set('colour', 'green');
+        expect(window.document.cookie).toEqual('testCookie={%22colour%22:%22green%22}');
```

Comment:
Can you reset this for each test? Multiple tests are sharing the same `window.document.cookie`